packages/api/src/db/comment.ts

```diff
@@ -0,0 +1,68 @@
+import {RichTextNode} from '../graphql/richText'
+
+export enum CommentStatus {
+  Approved = 'approved',
+  PendingApproval = 'pendingApproval',
+  PendingUserChanges = 'pendingUserChanges',
+  Rejected = 'rejected'
+}
+
+export enum CommentRejectionReason {
+  Misconduct = 'misconduct',
+  Spam = 'spam'
+}
+
+export enum CommentAuthorType {
+  Admin = 'admin',
```

Comment:
at the beginning i think we only have `Author, Team and normal user`

Reply:
do you think there is no need for this at the moment?